From 025f7a0695bf51317b23750c23f7c7824a0f9af1 Mon Sep 17 00:00:00 2001
From: Pierre Chambart <pierre.chambart@ocamlpro.com>
Date: Wed, 14 Jan 2015 14:41:08 +0100
Subject: [PATCH] Fix build with trunk

Internal global variable where renamed and a compatibility macro was
added in caml/compatibility.h
---
 lib/core_gc_stubs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/core_gc_stubs.c b/lib/core_gc_stubs.c
index 3e5431b..96d391e 100644
--- a/lib/core_gc_stubs.c
+++ b/lib/core_gc_stubs.c
@@ -1,4 +1,6 @@
 #include <caml/memory.h>
+#define CAML_NAME_SPACE
+#include <caml/compatibility.h>
 
 extern double caml_stat_minor_words;
 extern double caml_stat_promoted_words;
-- 
2.1.4

